frontend/packages/client/src/components/Community/CommunityEditorDetails.js

```diff
@@ -131,18 +131,12 @@ export const CommunityUsersForm = ({
           })}
         </div>
       )}
-      <div
-        className={`mt-2 is-flex is-align-items-centered ${
-          loadingUsers ? "is-disabled has-text-grey" : "cursor-pointer"
-        }`}
-        onClick={onAddAddress}
-        style={loadingUsers ? { opacity: 0.5 } : {}}
-      >
-        <Plus />{" "}
-        <span className="ml-2 small-text is-flex is-align-items-center">
-          Add{` ${addrType}`}
-        </span>
-      </div>
```

Comment:
<img width="297" alt="Screen Shot 2022-06-16 at 17 34 24" src="https://user-images.githubusercontent.com/7526740/174159141-ea0254ed-3a7f-4d67-971a-f7e63ea1dfd6.png">
also for authors but component is being reused so used only once